.travis.yml

```
@@ -2,7 +2,7 @@ sudo: false
 language: node_js
 
 node_js:
- - 8.10.0
+ - 12.0.0
```

Why not using the latest stable LTS version instead?
```suggestion
 - 12.18.1
```

https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md

---

update https://github.com/JetBrains/svg-sprite-loader/pull/403/commits/00f4fc2e23e902a4b2a0f387c515588ff4255d92